src/modules/page-load-errors.js

```diff
@@ -20,20 +20,12 @@
 	var actions = function () {
 		return {
 			pages: {
-				failedtoparse: function (key, data) {
-					
-				},
-				loaderror: function (key, data) {
-					
-				},
-				loadfatalerror: function (key, data) {
-					defaultLoadFatalError(key, data);
-				}
+				failedtoparse: defaultLoadFatalError(key, data),
```

Comment:
Remove this part to `(key, data)` it should on be `failedtoparse: defaultLoadFatalError,`